buildSrc/src/main/kotlin/com/bugsnag/android/gradle/JNILinkTablePlugin.kt

```diff
@@ -0,0 +1,67 @@
+package com.bugsnag.android.gradle
+
+import com.android.build.api.instrumentation.InstrumentationScope
+import com.android.build.api.variant.AndroidComponentsExtension
+import com.android.build.api.variant.Variant
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+import org.gradle.api.file.Directory
+import org.gradle.api.provider.Provider
+
+class JNILinkTablePlugin : Plugin<Project> {
```

Comment:
It would be nice to see a bit of code docs here to explain the plugin and rationale.